WalletWasabi.Fluent/Models/RecoveryWord.cs

```diff
@@ -0,0 +1,63 @@
+using ReactiveUI;
+using WalletWasabi.Gui.Validation;
+using WalletWasabi.Gui.ViewModels;
+using WalletWasabi.Models;
+
+namespace WalletWasabi.Fluent.Models
+{
+	public class RecoveryWord : ViewModelBase
+	{
+		private string _input;
+		private bool _isConfirmed;
+
+		public RecoveryWord(int index, string word)
+		{
+			Index = index;
+			Word = word;
+
+			this.ValidateProperty(x => x.Input, ValidateWord);
+		}
+
+		public string Input
+		{
+			get => _input;
+			set => this.RaiseAndSetIfChanged(ref _input, value);
+		}
+
+		public bool IsConfirmed
+		{
+			get => _isConfirmed;
+			set => this.RaiseAndSetIfChanged(ref _isConfirmed, value);
+		}
+
+		public int Index { get; }
+		public string Word { get; }
+
+		public void Reset()
+		{
+			Input = "";
+			IsConfirmed = false;
+		}
+
+		private void ValidateWord(IValidationErrors errors)
+		{
+			if (string.IsNullOrWhiteSpace(Input))
+			{
+				return;
+			}
+
+			if (Input == Word)
+			{
+				IsConfirmed = true;
+				return;
+			}
+
+			errors.Add(ErrorSeverity.Error, $"The input does not match the recovery word {Index}.");
```

Comment:
should be `The input does not match recovery word 1`

Reply:
done